Pre-processing/epochs.py

```diff
@@ -0,0 +1,69 @@
+import os
```

Comment:
This code is largely taken from: https://mne.tools/mne-realtime/_modules/mne/epochs.html.

Please make sure to cite any code you copy-paste.